src/index.js

```diff
@@ -157,7 +160,16 @@ export default {
                     await connector.connect();
                 }
 
+                this.tunnelConnectRetryCount = 0;
                 return connector;
+            })
+            .catch(error => {
+                this.tunnelConnectRetryCount++;
+
+                if (this.tunnelConnectRetryCount > MAX_TUNNEL_CONNECT_RETRY_COUNT)
+                    throw error;
+
+                return this._getConnector();
```

Comment:
`sauce-connect` is very slow, and the tunnel creation regularly failed at the end of the creation process, so it looks like the additional timeout is not needed.